config/manifests/bases/awx-operator.clusterserviceversion.yaml

```diff
@@ -308,6 +308,11 @@ spec:
         path: image_pull_secret
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: HostAliases for app containers
+        path: host_aliases
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:hidden
```

Comment:
if it's a `hidden` field type then this won't display in the UI. This needs to be a `text` type
```suggestion
        - urn:alm:descriptor:com.tectonic.ui:text
```

Reply:
host_aliases should be an array type，can it be represented by text type here？